Disease_db_2019-09-23.sql

```diff
@@ -0,0 +1,45 @@
+# ************************************************************
```

Comment:
You've got 2 .sql files in here, probably better to delete one, assuming one is obsolete